engines/director/score.cpp

```diff
@@ -180,6 +180,30 @@ void Score::loadArchive() {
 	setSpriteCasts();
 	loadSpriteImages(false);
 
+	// Try to load script name lists
+	if (_vm->getVersion() == 4) {
```

Comment:
Is it only D4? or D4+?

Reply:
Not sure! I haven't tried taking apart a D5 game yet

Reply:
Nice, D5 games share most of the data structures from D4, will open it up.